### PR TITLE
[json-rpc] add header forwarded into http request log

### PR DESCRIPTION
**Pull request by xli**
In case the request is forwarded from a proxy server, log forwarded for client address info

### CODE REVIEW

**Comment by libra-action** [comment]
**This PR may have modified JSON-RPC server code.**

Breaking changes policy:

1. Chaning JSON-RPC method signatures or removing/modifying fields
   in the response in /V1 will not be accepted.
2. Adding fields is ok.
3. Adding a new JSON-RPC method is ok.

Please ensure you have also done the following to update the docs:

1. Update json-rpc/API-CHANGELOG.md.
2. Add/update type/method documentation under json-rpc/docs.

File: json-rpc/src/runtime.rs [hunk 1]
```diff
@@ -122,6 +124,10 @@ pub fn bootstrap(
                 referer: info.referer(),
                 user_agent: info.user_agent(),
                 elapsed: info.elapsed(),
+                forwarded: info
+                    .request_headers()
+                    .get(header::FORWARDED)
```
Comment (mgorven) [inline json-rpc/src/runtime.rs:129]: What's the header name?

Comment (xli) [reply json-rpc/src/runtime.rs:129]: forwarded

Comment (xli) [reply json-rpc/src/runtime.rs:129]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded

Comment (bmwill) [reply json-rpc/src/runtime.rs:129]: So we know that HAproxy sets this header with the actual IP of the remote?

Comment (mgorven) [reply json-rpc/src/runtime.rs:129]: I'll get that part done.

**Review by mgorven — APPROVED** [review]

**Pull request closed without merging**